apps/els_lsp/src/els_compiler_diagnostics.erl

```diff
@@ -76,6 +77,7 @@ source() ->
 
 -spec on_complete(uri(), [els_diagnostics:diagnostic()]) -> ok.
 on_complete(Uri, Diagnostics) ->
+  ?MODULE:telemetry(Uri, Diagnostics),
```

Comment:
Nit: any reason for the fully qualified call

Reply:
So that we can mock it in the tests.

Reply:
In that case it should be enough to export the function. Unless we are mocking both `telemetry/2` and `on_complete/2` (in which case I think meck requires the fully qualified call to be able to work.

Reply:
If I call the local version only, it never sees the mocked one.  This is the same reason `?MODULE:publish(...` is called in https://github.com/erlang-ls/erlang_ls/blob/d50da89ebb01e885c06d82b5f338205573f0fc45/apps/els_lsp/src/els_diagnostics_provider.erl#L61

Reply:
And I was the one who did that 😅 I guess it's because of the `passthrough` to the module itself...